pages/api/class/[id].ts

```diff
@@ -33,8 +35,27 @@ export default async function handle(
         }
         ResponseUtil.returnOK(res, deletedClass);
         return;
+    } else if (req.method == "PUT") {
+        // validate updated class body
+        const classInput = req.body as ClassInput;
+        const validationError = validateClassData(classInput);
+        if (validationError.length !== 0) {
+            ResponseUtil.returnBadRequest(res, validationError.join(", "));
+            return;
+        }
+        // obtain class id
+        const { id } = req.query;
```

Comment:
I think we should perhaps validate it prior to passing it into `updateClass`. See below for reference:

https://github.com/uwblueprint/social-diversity-for-children/blob/9a6325a317bda14b4033bb75ac94c53592db5c6c/pages/api/enroll/child.ts#L34-L44

Reply:
Also, isn't `id` already being retrieved from req.query on line 18?

Reply:
We can do this validation right after line 18 as for both GET and PUT, we want to make sure `id` is a number.

Reply:
After you make that change, you can update function signature for `updateClass` to accept id as a number instead!